migrations/versions/e02fe8fde71e_unit_contact_email_non_nullable.py

```diff
@@ -0,0 +1,33 @@
+"""unit_contact_email_non_nullable
+
+Revision ID: e02fe8fde71e
+Revises: 
+Create Date: 2024-12-02 13:08:43.073488
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e02fe8fde71e"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
In the upgrade, also add values to rows where the contact_email is not filled in. See other migrations where similar things have been done.